addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -287,7 +287,7 @@ export class LinkPlugin extends Plugin {
                 isAvailable: () =>
                     this.dependencies.selection.getSelectionData().documentSelectionIsInEditable,
             }
-        );
+        ) || (() => {});
```

Comment:
Wouldn't it be better to write `this.unregisterLinkCommandCallback?.();` at call site instead so that you don't have to write an empty function?